persistence/src/main/scala/hmda/persistence/processing/ProcessingMessages.scala

```diff
@@ -25,5 +25,6 @@ object ProcessingMessages {
   case class ParsingCompletedWithErrors(submissionId: SubmissionId) extends Event
   case class ValidationCompleted(submissionId: SubmissionId) extends Event
   case class ValidationCompletedWithErrors(submissionId: SubmissionId) extends Event
+  case class Persisted() extends Event
```

Comment:
I'm not clear on what empty parens do. Could `Persisted` work, rather than `Persisted()`? If there's a benefit to one or the other, what is it?